test/unit/kubernetes-deploy/kubernetes_resource_test.rb

```diff
@@ -399,7 +399,7 @@ def dummy_events(start_time)
         count: 3,
         last_seen: start_time + 3.seconds,
         reason: "FailedSync",
-        message: <<~STRING
+        message: <<~STRING,
```

Comment:
Is this really where the `,` goes?

Reply:
Yep--it's like when you chain a method on a heredoc, this is where it goes (`<<~STRING.strip` for example). I agree this comma looks stupid though. 😞